example/glue/ConfigurationDelegate.hpp

```diff
@@ -175,6 +175,16 @@ class MM_ConfigurationDelegate
 	 */
 	MM_GCPolicy getGCPolicy() { return _gcPolicy; }
 
+#if defined(J9VM_OPT_CRIU_SUPPORT)
+	/**
+	 * If the checkpointgcthread is specified by the user, it is verified
+	 * to be not larger than the number of gc threads. If it is not specified
+	 * by the user, it is adjusted to match the number of gc threads.
+	 * @param[in] env The environment for the calling thread.
+	 */
```

Comment:
Add a new line for clarity, and param/return lines don't need a period/full-stop.
```suggestion
	 * by the user, it is adjusted to match the number of gc threads.
	 *
	 * @param[in] env The environment for the calling thread
	 */
```